server/controllers/project.controller.js

```diff
@@ -3,7 +3,7 @@ const Project = require('../models/project.model');
 
 module.exports = {
     findAll: (req, res) => {
-        Project.find().sort('dueDate')
+        Project.find({creator: req.body.id}).sort('dueDate')
```

Comment:
Hi Sara, shouldn't we create a new function to find by ID instead of changing  the findAll function?

Reply:
Nope, this is getting all projects based on the 'creator' which is the user ID for the user that created the task.

Reply:
that makes sense :)